nvi-commons/src/main/java/no/sikt/nva/nvi/common/model/business/Candidate.java

```diff
@@ -21,6 +26,42 @@ public record Candidate(URI publicationId,
                         List<ApprovalStatus> approvalStatuses,
                         List<Note> notes) {
 
+    public Candidate {
+    }
+
+    public AttributeValue toDynamoDb() {
+        Map<String, AttributeValue> map = new HashMap<>();
+        map.put("publicationId", AttributeValue.fromS(publicationId.toString()));
```

Comment:
Maybe create constants for these string values, since you use them in both toDynamoDb and fromDynamoDb methods